docs/getting-started/v3-migration.md

```diff
@@ -26,6 +26,7 @@ Chart.js is no longer providing the `Chart.bundle.js` and `Chart.bundle.min.js`.
 ### Customizability
 
 * `custom` attribute of elements was removed. Please use scriptable options
+* The `zeroLine*` features of axes were removed.
```

Comment:
I would say either "options" (plural) or "feature" (singular). In my mind it's one feature with a few corresponding options